src/tools/immutable_object_cache/Types.h

```diff
@@ -85,12 +85,13 @@ class ObjectCacheReadData : public ObjectCacheRequest {
   uint64_t read_len;
   uint64_t pool_id;
   uint64_t snap_id;
+  uint64_t object_size;
```

Comment:
Nit: default initialize in case it's not decoded.

Reply:
Does default initialization mean defining a default constructor function for initialization?

Reply:
You can just do this:
```suggestion
  uint64_t object_size = 0;
```